CHANGES.md

```diff
@@ -2,6 +2,14 @@
 
 ## __NEXT__
 
+* ancestral, translate: Major improvements to how we parse GFF and GenBank reference files. [#1351][] (@jameshadfield)
```

Comment:
https://github.com/nextstrain/augur/pull/1344#discussion_r1416477632 😉

Reply:
How do you normally categorise changes? e.g. would 

> For GenBank files we now require the (mandatory) source feature to be present

be a breaking change and therefore a "Major change". Similarly for things like

> For GFF files, we extract the genome/sequence coordinates by inspecting the sequence-region pragma, region type and/or source type. This information is now required.

Reply:
I'd classify anything that potentially breaks an existing interface as a major change. In following description, the first sentence describes a new feature since it's adding some new functionality that didn't exist (although you could interpret this as fixing a bug in our GFF parsing 😄). The second sentence makes the change a major change since the new requirement for the nucleotide coordinates could break an existing workflow.

> For GFF files, we extract the genome/sequence coordinates by inspecting the sequence-region pragma, region type and/or source type. This information is now required.

Reply:
Thanks - updated via [force-push](https://github.com/nextstrain/augur/compare/454c53ccb51fe933a2a1bd0ac4dbbfa8a6025d20..d11ba4e65041e865b2da0400407016f6bc295be9)

Reply:
Thanks! Good general advice. Updated changelog accordingly.